lib/ie-contract.js

```diff
@@ -22,5 +23,16 @@ export const createMeridianContract = async (contractAddress = SparkImpactEvalua
     SparkImpactEvaluator.ABI,
     provider
   )
+
+  try {
+    await pTimeout(ieContract.currentRoundIndex(), {
+      milliseconds: 10_000
+    })
+  } catch (err) {
+    console.error(`Failed to connect to contract via ${RPC_URL}`)
+    console.error(err)
+    process.exit(1)
+  }
```

Comment:
Should we send this error to Sentry?

Also, why `process.exit`? Isn't it enough to (re)throw the error and let the "unhandled rejection" handler exit the process? That way, the error should be automatically captured & sent to Sentry.

Reply:
Good idea! I hadn't considered just throwing